go.mod

```diff
@@ -60,7 +60,7 @@ require (
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.8.0
 	github.com/tiancaiamao/appdash v0.0.0-20181126055449-889f96f722a2
-	github.com/tikv/client-go/v2 v2.0.1-0.20220809034808-2ed2113d1090
```

Comment:
Please remember to run ```gazelle``` for sync dependency to bazel.

Reply:
Done